learntools/machine_learning/nbs/ch2.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This exercise will test your ability to read a data file and understand statistics about the data.\n",
+    "\n",
+    "In later exercises, you will techniques to filter the data, build a machine learning model, and iteratively improve your model.\n",
+    "\n",
+    "The course gives examples using data from Melbourne. To ensure you can apply these techniques on your own, you will have to apply them to a new dataset (with house prices from Iowa).\n",
+    "\n",
+    "The exercises use a \"notebook\" coding environment.  In case you are unfamiliar with notebooks, we have a [90-second intro video](https://www.youtube.com/watch?v=4C2qMnaIKL4).\n",
+    "\n",
+    "# Exercises\n",
+    "\n",
+    "Run the following cell to set up code-checking, which will verify your work as you go."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Set up code checking\n",
+    "from learntools.core import binder\n",
+    "binder.bind(globals())\n",
+    "from learntools.machine_learning.ex2 import *\n",
+    "print(\"Setup Complete\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: Loading Data\n",
+    "Read the file into a Pandas DataFrame called `home_data`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "# Path of the file to read\n",
+    "iowa_file_path = '../input/house-prices-advanced-regression-techniques/train.csv'\n",
+    "\n",
+    "# Fill in the line below to read the file into a variable home_data\n",
+    "#home_data = _\n",
+    "\n",
+    "# Call line below with no argument to check that you've loaded the data correctly\n",
+    "step_1.check()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Lines below will give you a hint or solution code\n",
+    "# step_1.hint()\n",
+    "# step_1.solution()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Review The Data\n",
+    "Use the command you learned to view summary statistics of the data. Then fill in variables to answer the following questions"
```

Comment:
The example code in the tutorial does `print(df.describe())`. If they do that on the Iowa data, they get an abbreviated output with only 4 columns (which doesn't include the columns they need to answer this question). But if they just type `df.describe()`, they'll get a nice horizontally-scrolling table with all the columns. I'd suggest changing the tutorial example to not use print.